### ISSUE TITLE
Ninety degrees clockwise rotation

### ISSUE THREAD

**Issue by estebansotoara** [ruslanskorb/RSKImageCropper#81]
Hi
I added a functionality to rotate the image by ninety degrees clockwise, the functionality is disabled by default and can be enabled setting the parameter '''fixedRotationEnabled''' to YES, this shows a button that allows the rotation. Also added a few test for this. 

I have a fork with the changes (https://github.com/estebansotoara/RSKImageCropper/tree/ninetyDegreesRotation), you can check it out and if you like it I can make a PR

BTW: there are 8 test that are failing, checked on a clean copy of master and are failing there too so idk if it is a known issue or just failing to me for some strange reason

**Comment by ruslanskorb** [comment]
Hi Esteban,

Thank you for your efforts to make RSKImageCropper better!
I'm glad to see that you have added tests :+1: 

My main idea for RSKImageCropper is to keep it simple. I think ninety degrees rotation can be useful for RSKImageCropper in some cases, but mostly it will not be used. It can be a good idea to implement this functionality in subclass of RSKImageCropViewController and use specially for your purpose. Additionally I would like to suggest you to use an image for the button 'Rotate' instead of the title. It will be more clear for the end-user how rotation works.

A few words about the tests. Currently in the repository we have valid snapshots only for iPhone 4s / iOS 8.1. To add additional snapshots locally you need to replace `haveValidSnapshot` by `recordSnapshot`, run tests once, replace `recordSnapshot` by `haveValidSnapshot` and run tests again. All tests should pass successfully.

**Comment by estebansotoara** [comment]
Hi ruslanskorb, thanks for your answer.

You are right, this can be achieved without pain using a subclass, I did it in case someone need it just out of the box. As for the image for the rotate button I actually thought about using one but my skills in icon design are not very high and didn't just wanted to grab someone else's image so I decided to leave it as a text, other possibility could be leave the possibility to set an image from outside and default to text if the image was not provided. I'll leave the fork alive anyway in case someone wants to use it.

And good to know about the tests, that was driving me crazy I had a vague idea about what's happening but could't figure it out exactly how to make them work.
